source/python/sql.txt

```diff
@@ -0,0 +1,23 @@
+Before you can run SQL queries against your DataFrame, you need to
+register a temporary table.
+
+The following example registers a temporary table called ``temp``,
+then uses SQL to query for records in which the ``type`` field
+contains the letter ``e``:
+
+.. code-block:: python
+
+   df.createOrReplaceTempView("temp")
+   some_fruit = spark.sql("SELECT type, qty FROM temp WHERE type LIKE '%e%'")
+   some_fruit.show()
+
+In the ``pyspark`` shell, the operation prints the following output:
```

Comment:
issue (no action at this time): Is there an MQL equivalent they could use to also achieve this? It might be nice to show it to contrast, and highlight that it would be pushed down.